projects/1.16/assets/techreborn/techreborn/lang/zh_cn.json

```
@@ -1 +1,984 @@
+  "_comment18": "信息",
+  "techreborn.message.missingmultiblock": "多方块结构缺失",
+  "techreborn.message.setTo": "设置为",
+  "techreborn.message.in": "在",
+  "techreborn.message.noCoordsSet": "没有设置坐标",
+  "techreborn.message.coordsHaveBeen": "坐标以设置为",
+  "techreborn.message.cleared": "清除",
+  "techreborn.message.detects": "检测",
+  "techreborn.message.alarm": "切换到：",
+  "techreborn.message.allPlayers": "所有玩家",
+  "techreborn.message.onlyOtherPlayers": "仅其他玩家",
+  "techreborn.message.onlyYou": "仅自己",
+  "techreborn.message.daygen": "阳光直射面板",
+  "techreborn.message.nightgen": "阳光减少的面板",
+  "techreborn.message.zerogen": "被天空遮挡的面板",
+  "techreborn.message.info.item.techreborn.overclocker_upgrade": "能量换取性能",
+  "techreborn.message.info.item.techreborn.transformer_upgrade": "提高能量等级",
+  "techreborn.message.info.item.techreborn.energy_storage_upgrade": "增加能量存储量",
+  "techreborn.message.info.item.techreborn.superconductor_upgrade": "提高能源流速率",
+  "techreborn.message.info.block.techreborn.basic_solar_panel": "从阳光中生产能量",
+  "techreborn.message.info.block.techreborn.advanced_solar_panel": "从阳光中生产能量",
+  "techreborn.message.info.block.techreborn.industrial_solar_panel": "从阳光中生产能量",
+  "techreborn.message.info.block.techreborn.ultimate_solar_panel": "从阳光中生产能量",
+  "techreborn.message.info.block.techreborn.quantum_solar_panel": "从阳光中生产能量",
+  "techreborn.message.info.block.techreborn.creative_solar_panel": "无限能源，创造",
+  "techreborn.message.info.block.techreborn.fusion_control_computer": "聚变反应堆的核心",
+  "techreborn.message.info.block.techreborn.creative_quantum_chest": "存储几乎无限的单个物品",
+  "techreborn.message.info.block.techreborn.adjustable_su": "可调储能\n可配置",
+  "techreborn.message.info.block.techreborn.interdimensional_su": "高层能量存储\n同步无线能量传输",
+  "techreborn.message.info.block.techreborn.low_voltage_su": "基本级能量储存",
+  "techreborn.message.info.block.techreborn.medium_voltage_su": "中级能量储存",
+  "techreborn.message.info.block.techreborn.high_voltage_su": "高级能量储存",
+  "techreborn.message.info.block.techreborn.lv_transformer": "中低能量级",
+  "techreborn.message.info.block.techreborn.mv_transformer": "中高能量级",
+  "techreborn.message.info.block.techreborn.hv_transformer": "极高能量级",
+  "techreborn.message.info.block.techreborn.ev_transformer": "至高能量级",
+  "techreborn.message.info.block.techreborn.alarm": "接收到红石信号时发出声音\n可配置",
+  "techreborn.message.info.block.techreborn.lamp_incandescent": "通电时提供亮度",
+  "techreborn.message.info.block.techreborn.lamp_led": "通电时提供亮度",
+  "techreborn.message.info.block.techreborn.player_detector": "用于检测玩家的释放红石信号的方块，可配置",
+  "techreborn.message.info.block.techreborn.alloy_smelter": "用锭或粉末制成合金",
+  "techreborn.message.info.block.techreborn.resin_basin": "自动从橡胶树上收集橡胶\n直接放在橡胶树旁\n可在下方用箱子存储橡胶",
+  "techreborn.message.info.block.techreborn.assembly_machine": "更高效地组装组件",
+  "techreborn.message.info.block.techreborn.auto_crafting_table": "根据能量和资源自动合成物品",
+  "techreborn.message.info.block.techreborn.chemical_reactor": "用两种化学品构成新的产品",

```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
  "techreborn.message.info.block.techreborn.chemical_reactor": "用两种化学品相互反应，形成新产品。",

```